src/parser/smt2/smt2.cpp

```diff
@@ -293,6 +293,12 @@ void Smt2::addTheory(Theory theory) {
     defineType("String", getExprManager()->stringType());
     defineType("RegLan", getExprManager()->regExpType());
     defineType("Int", getExprManager()->integerType());
+
+    defineVar("re.all",
```

Comment:
Should this be a part of addStringOperators?

Reply:
Yeah, I suppose that would be better.